prisma/migrations/20230426123806_delete_review/migration.sql

```diff
@@ -0,0 +1,58 @@
+-- CreateTable
+CREATE TABLE "user" (
+    "email" VARCHAR(50) NOT NULL,
```

Comment:
what is this migration for?

Reply:
I hadn't run `npx prisma migrate dev`

Reply:
I use npx prisma db push. This way we don't have to deal with migrations. If i remember correctly migrations are used when there's already data in prod and prod is already being used, right @idiglove?